apps/estuary/mix.exs

```diff
@@ -28,13 +28,15 @@ defmodule Estuary.MixProject do
 
   defp deps do
     [
-      {:credo, "~> 1.1", only: :dev, runtime: false},
+      {:credo, "~> 1.1", only: [:dev, :test], runtime: false},
```

Comment:
you shouldn't need credo in the test environment. if you're getting a warning about that because of another app in the pipeline, try deleting it from test in that app as well.

Reply:
Good call